### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by qwemaze** [katzer/cordova-plugin-local-notifications#655]
I'm using cordova5.1.1 and crosswalk.

```
...n\LocalNotification.java:495: error: cannot find symbol
                    webView.evaluateJavascript(js, null);
                           ^
  symbol:   method evaluateJavascript(String,<null>)
  location: variable webView of type CordovaWebView
C:\Users\qweli_000\Documents\Repositories\puzzle\platforms\android\src\de\appplant\cordova\plugin\localnotification\LocalNotification.java:492: error: cannot find symbol
        webView.post(new Runnable(){
               ^
  symbol:   method post(<anonymous Runnable>)
  location: variable webView of type CordovaWebView
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors
:compileArmv7DebugJava FAILED
```

**Comment by sirius007** [comment]
I have the same issues

**Comment by danielefenix** [comment]
+1

**Comment by papasmurph** [comment]
Yup, same issue here. Works fine for 3.7.0.

**Comment by qwemaze** [comment]
Fixed by installing this https://github.com/floatinghotpot/cordova-plugin-android-support-v4
and then by fetching plugin from repo https://github.com/katzer/cordova-plugin-local-notifications

**Comment by papasmurph** [comment]
Not applicable to me, as I use PhoneGap Build.

**Comment by landall** [comment]
android.support.v4 21.0.1 "Android Support v4"

installed but not make effect.....

I just comment some lines of the source however make no effect to Phonegap build...

**Comment by FlossyWeb** [comment]
+1 same here

**Comment by colonelchlorine** [comment]
+1 same

**Comment by colonelchlorine** [comment]
So, I checked LocalNotification.java in the current master state and line 495 doesn't have this issue. I think when we're doing a `cordova plugin add ...` it's just getting it from another resource.

I just cloned this repo into my root of the cordova project, and then added *that* as a plugin.

```
git clone https://github.com/katzer/cordova-plugin-local-notifications
cordova plugin add cordova-plugin-local-notifications/
```

Now it builds fine.

**Comment by mladenp** [comment]
@colonelchlorine yup you are right, works when you clone locally and then add plugin. It doesn't seem to have anything to do with android support lib.

**Comment by 255kb** [comment]
Same build error here, and works fine when cloned locally

**Comment by katzer** [comment]
@qwemaze Fixed with v0.8.2
